week-1/implement/max.js

```diff
@@ -0,0 +1,39 @@
+// Manual check
+// function max(input){
+//     if(input.length == 0){
+//         return -Infinity;
+//     }
+
+//     if(input.length == 1){
+//         return input[0];
+//     }
+//     let maxNum = input[0]; 
+//     for (let i=0; i<input.length; i++){
+//         if(typeof input[i] !== 'string'){
+//             maxNum = input[i]
+//             break;
+//         }
+//     }
+
+//     for (let i=1; i<input.length; i++){
+//         if(input[i] > maxNum){
+//             maxNum = input[i];
+//         } 
+//     }
+//     return maxNum;
+// }
+
+
+function max(input){
+    let output = [];
```

Comment:
This function is not handling the case for -Infinity, can you add that bit please?
I think you had in the commented code above so you may copy that into the working function and get rid of the commented